### DISCUSSION TITLE
Üssünü Alma

### DISCUSSION THREAD

**Discussion by HabilCcabbar** [sonsuzus/sonsuzus#345] [Soru Cevap]
Verilen iki sayı A ve B. A^B'nin değerini hesaplayınız. Çıktı çok büyük olabilir, bu nedenle cevabı modül 10^9+7'ye uygun yazdırın.

Giriş:
Giriş, boşlukla ayrılmış iki tam sayı A ve B içerir.

Çıktı:
A^B'nin 10^9+7 modülüne uygun halini yazdırın

Kısıtlamalar:
Test Dosyası 1'den 5'e:
1<=A<=10
1<=B<=10

Test Dosyası 6'dan 10'a:
1<=A<=10^6
1<=B<=10^10

Örnek Girdi:
2 5
Örnek Çıktı:
32

Orijinal Soru: https://www.hackerearth.com/practice/math/number-theory/basic-number-theory-1/practice-problems/algorithm/calculate-the-power/

**Comment by MASahin223** [comment]
```python
girdi = input("")
sayac = 0
sayac2 = -1
sayi2 = 0
x = 0
y = 0
deger = 0
for i in girdi:
    if i == " ":
        sayi2 = 1
        sayac = 0
    if sayi2 == 0 and sayac == 0:
        x = i
    if sayac > 0:
        x = x+i
    if sayi2 == 1 and sayac2 == 0:
        y = i
    if sayac2 > 0:
        y = y+i
    if sayi2 == 0:
        sayac = sayac + 1
    if sayi2 == 1:
        sayac2 = sayac2 + 1
x = int(x)
y = int(y)
deger = x
if (y-1) != 0 and (y-1) != -1:
    for i in range(y-1):
        deger = deger * x
if (y-1) == -1:
    deger = 1
print(deger)
```
